Kudu.Contracts/Functions/MasterKeyOperation.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+
+namespace Kudu.Core.Functions
+{
+    public class MasterKeyOperation : IKeyOperation
+    {
+        public JObject GenerateKeyJson(string masterKey)
+        {
+            JObject hostJson = JObject.Parse($"{{\"masterKey\":{{\"name\":\"master\",\"value\":\"{masterKey}\",\"encrypted\": true }}}}");
```

Comment:
there is no previous behavior in kudu since it did not return keys in "host.json". looking at the [CanSerialize()](https://github.com/fabiocav/azure-webjobs-sdk-script/blob/31cd7632375a65b290c08dd80b7a5a9fbdfb71e3/src/WebJobs.Script.WebHost/Security/ScriptSecretSerializerV1.cs#L30-L33) function, it seems that if "host.json" does not have a "functionKeys" JArray, it is considered as malformatted...but we don't check whether if the JArray is empty, do you want to create an empty "functionKeys" JArray or an encrypted key name "default"